src/pages/home/report/ContextMenu/ReportActionContextMenu.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+const contextMenuRef = React.createRef();
+
+/**
+ * Show the ReportActionContextMenu modal popover.
+ *
+ * @param {Object} [event] - A press event.
+ * @param {string} [selection] - A copy text.
+ * @param {Element} contextMenuAnchor - popoverAnchor
+ * @param {Number} reportID - Active Report Id
+ * @param {Object} reportAction - ReportAction for ContextMenu
+ * @param {String} draftMessage - ReportAction Draftmessage
+ * @param {Function} [onShow=() => {}] - Run a callback when Menu is shown
+ * @param {Function} [onHide=() => {}] - Run a callback when Menu is hidden
+ */
+function showContextMenu(
+    event,
+    selection,
+    contextMenuAnchor,
+    reportID,
+    reportAction,
+    draftMessage,
+    onShow = () => {},
+    onHide = () => {},
+) {
+    if (!contextMenuRef.current) {
+        return;
+    }
+    contextMenuRef.current.showContextMenu(
+        event,
+        selection,
+        contextMenuAnchor,
+        reportID,
+        reportAction,
+        draftMessage,
+        onShow,
+        onHide,
+    );
+}
+
+/**
+ * Hide the ReportActionContextMenu modal popover.
+ * Hides the popover menu with an optional delay
+ * @param {Boolean} shouldDelay - whether the menu should close after a delay
+ * @param {Function} [onHideCallback=() => {}] - Callback to be called after Context Menu is completely hidden
+ */
+function hideContextMenu(shouldDelay, onHideCallback = () => {}) {
+    if (!contextMenuRef.current) {
+        return;
+    }
+    if (!shouldDelay) {
+        contextMenuRef.current.hideContextMenu(onHideCallback);
+
+        return;
+    }
+    setTimeout(() => contextMenuRef.current.hideContextMenu(onHideCallback), 800);
+}
+
+function hideDeleteModal() {
+    if (!contextMenuRef.current) {
+        return;
+    }
+    return contextMenuRef.current.hideDeleteModal();
```

Comment:
NAB, is there some reason we are returning this function?

Reply:
Sorry, It must have been left unintentionally. Only isActiveReportAction needs return value. I will update it.